profiles/multisite_drupal_communities/modules/custom/multisite_drupal_features_set_og/multisite_drupal_features_set_og.info

```diff
@@ -282,7 +282,7 @@ feature_sets[splash_screen][disable][] = splash_screen
 
 feature_sets[nexteuropa_dgt_connector][title] = 'Nexteuropa DGT Connector'
 feature_sets[nexteuropa_dgt_connector][description] = 'Provides services to send translations to DGT.'
-feature_sets[nexteuropa_dgt_connector][documentation] = 'https://github.com/ec-europa/platform-dev/tree/master/profiles/common/modules/features/nexteuropa_dgt_connector/README.md'
+feature_sets[nexteuropa_dgt_connector][documentation] = 'https://github.com/ec-europa/platform-dev/tree/master/profiles/common/modules/custom/nexteuropa_dgt_connector/README.md'
```

Comment:
master is no longer available, i suggest changing it to release-2.6/profiles/...